### PR TITLE
[PCEE RN] Lagrange Update 1

### PR DESCRIPTION
**Pull request by Pubs-MV**
[PCEE RN] Lagrange Update 1

### CODE REVIEW

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+
+* Fixed versions enriched for fixed vulnerabilities.
+
+* New PRISMA-IDs have increased 131% since the Kepler major release.
+
+* Fast addition of CVEs (pre-filled CVEs).
++
+CVEs were added to the Intelligence Stream on an average of 13 days before they were analyzed in the NVD.
+As an example, a Kubernetes CVE (CVE-2022-3172) was published on September 16, 2022. The CVE was added to the Prisma Cloud Intelligence stream on September 19, 2022. And at this time in December 2022, the CVE is still reserved in MITRE and not analyzed in NVD. 
+2+|*New Features in the Core Platform*
```
Comment (grevach) [inline cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:41]: @divyabhushan why do we have 2 titles for new features in Core? the second is on line 78

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+|AWS recommends the use of a regional STS endpoint over the use of the global STS endpoint `sts.amazonaws.com`.
+When onboarding your AWS cloud account, you can now use a regional `sts.REGION.amazonaws.com` STS endpoint.
+Then, your deployed Defenders don't need to access the global STS endpoint.
+Defenders can get the STS token from the regional STS endpoint to perform scans such as registry scans.
+To enable regional STS endpoints, refer to the https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html[AWS documentation].
+
+//GH#36695 - How to create a table within a row?
+
+//GH#36697
+|*Support to Generate Vulnerability Reports by Package*
```
Comment (grevach) [inline cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc]: @divyabhushan in the title we should also mention risk factors
```suggestion
|*Support to Generate Vulnerability Reports by Package and Risk Factors*
```

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+
+You can view the scan results on the mounted volume and on "Monitor > Runtime > Image analysis sandbox". 
+In this example the output of the 3rd party testing tool will be written to the `/opt/sandbox_testing_tools/output.txt file` on the sandbox host.
+
+// CWP-41281 (Update1)
+|*Twistcli for ARM64 Mac*
+|twistcli is now supported on ARM64 Mac machines.
+
+Download the ARM64 Mac-compatible version of twistcli from "Manage > System > Utilities", or using the API `/util/osx/arm64/twistcli`.
+
```
Comment (grevach) [inline cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:316]: @divyabhushan we are missing 2 features here, that are a part of Lagrange Update 1 (they are mentioned in the PCCE release notes):
1. Filter Defenders by TAS Foundation ID https://redlock.atlassian.net/browse/CWP-42365
2. Support for Talos Linux - container vulnerabilities and compliance https://redlock.atlassian.net/browse/CWP-42198

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc [hunk 4]
```diff
@@ -16,25 +16,25 @@ The list of addressed issues are not cumulative; only the issues that are fixed
 |Description
```
Comment (grevach) [inline cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc:17]: @Pubs-MV @divyabhushan are we getting these updates on the known issues article for the Compute Edition as well?

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc [hunk 5]
```diff
@@ -16,25 +16,25 @@ The list of addressed issues are not cumulative; only the issues that are fixed
...
-+++<draft-comment></draft-comment>+++added update 2
+//added Kepler Update 2
 |The userinput:[--tarball] option in twistcli does not scan for compliance checks. Currently, only vulnerabilities are detected successfully.
 
 
 |*—* 
 
-+++<draft-comment>25370</draft-comment>+++
+// #25370
 |For custom compliance checks for Kubernetes and OpenShift on CRI-O, if you have configured *Reported results* to show both passed and failed checks, in the event that a check doesn’t run, Prisma Cloud still reports it as passed.
```
Comment (grevach) [inline cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc]: @Pubs-MV @divyabhushan 
This now works, see the last comment here https://redlock.atlassian.net/browse/CWP-25370?focusedCommentId=606551
We can remove it from the known issues

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc [hunk 6]
```diff
@@ -0,0 +1,682 @@
...
+* PUT, api/vVERSION/settings/registry
+
+|*DISA STIG Scan Findings and Justifications*
+|Every https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-compute-edition-public-sector/Release_Findings[release], we perform an SCAP scan of the Prisma Cloud Compute Console and Defender images. The process is based upon the U.S. Air Force’s Platform 1 "Repo One" OpenSCAP scan of the Prisma Cloud Compute images. We compare our scan results to IronBank’s latest approved UBI8-minimal scan findings. Any discrepancies are addressed or justified.
+
+|===
+
+[#id-addressed-issues]
+=== Addressed Issues
+
```
Comment (grevach) [inline cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:564]: @divyabhushan missing this bug for the addressed issues, it was also fixed for update 1 https://redlock.atlassian.net/browse/CWP-42634
CVSS3 score wasn't displayed for Harbor 2.5 and up (there was a change on Harbor's side), so we fixed it to handle their change.

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc (hunk 5 above)
Comment (divyabhushan-okta) [reply cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc]: Done. Removed it. Thanks

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc (hunk 1 above)
Comment (divyabhushan-okta) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:41]: @grevach Thanks for pointing that out. I fixed this in the latest commit.

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc (hunk 2 above)
Comment (divyabhushan-okta) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc]: Done.

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc (hunk 3 above)
Comment (divyabhushan-okta) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:316]: @grevach Added these two features. Thanks.

Comment (rcaballeromx) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:316]: @grevach I've added a mention on sorting by foundation for TAS on the Windows support RN and think that we should include it on the TAS defender deployment flow. WDYT?
I've also added the limitations regarding Talos Linux. Once support is more robust, we might need to create a deployment flow for it too, but IMO we don't need to do it now.
Thanks @divyabhushan and let me know if there are any other RN updates you need from me.

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc (hunk 4 above)
Comment (Pubs-MV) [reply cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc:17]: @divyabhushan please confirm that we did.

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc (hunk 6 above)
Comment (Pubs-MV) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:564]: I added CWP-42634 to this RN. Please add it to the PCCE addressed issues too, @divyabhushan

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc (hunk 4 above)
Comment (divyabhushan-okta) [reply cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc:17]: @grevach Yes, we added these updates under known issues for PCCE [here](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/767/files#diff-8ea760a8e7be938f27da286dd3f64a7461e10e7ba82ca876e3732a2ef336c32b)

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc (hunk 6 above)
Comment (divyabhushan-okta) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:564]: Thanks @Pubs-MV. I added the addressed issue in PCCE [here](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/767/commits/6517d3c4a9423b6e1eea656c5b57efc68579e284).

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc (hunk 3 above)
Comment (grevach) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc:316]: @rcaballeromx the filter by Foundation is not related specifically to the TAS windows support. We can add it to the TAS defender deployment flow
Regarding Talos, where did you add the limitations? I'm not sure it should be in the release notes but somewhere in the admin guide, maybe where we declare support for Talos in the sys requirements

**Pull request merged**